translib/transformer/xfmr_sflow.go

```diff
@@ -0,0 +1,748 @@
+	SAMPLING_SFLOW_STATE_ENABLED                = "/openconfig-sampling-sflow:sampling/sflow/state/enabled"
+	SAMPLING_SFLOW_STATE_POLLING_INT            = "/openconfig-sampling-sflow:sampling/sflow/state/polling-interval"
+	SAMPLING_SFLOW_STATE_AGENT                  = "/openconfig-sampling-sflow:sampling/sflow/state/agent"
+	SAMPLING_SFLOW_COLS                         = "/openconfig-sampling-sflow:sampling/sflow/collectors"
+	SAMPLING_SFLOW_COLS_COL                     = "/openconfig-sampling-sflow:sampling/sflow/collectors/collector"
+	SAMPLING_SFLOW_COLS_COL_CONFIG              = "/openconfig-sampling-sflow:sampling/sflow/collectors/collector/config"
+	SAMPLING_SFLOW_COLS_COL_STATE               = "/openconfig-sampling-sflow:sampling/sflow/collectors/collector/state"
+	SAMPLING_SFLOW_INTFS                        = "/openconfig-sampling-sflow:sampling/sflow/interfaces"
+	SAMPLING_SFLOW_INTFS_INTF                   = "/openconfig-sampling-sflow:sampling/sflow/interfaces/interface"
+	SAMPLING_SFLOW_INTFS_INTF_CONFIG            = "/openconfig-sampling-sflow:sampling/sflow/interfaces/interface/config"
+	SAMPLING_SFLOW_INTFS_INTF_CONFIG_SAMPL_RATE = "/openconfig-sampling-sflow:sampling/sflow/interfaces/interface/config/sampling-rate"
+	SAMPLING_SFLOW_INTFS_INTF_CONFIG_ENABLED    = "/openconfig-sampling-sflow:sampling/sflow/interfaces/interface/config/enabled"
+	SAMPLING_SFLOW_INTFS_INTF_STATE             = "/openconfig-sampling-sflow:sampling/sflow/interfaces/interface/state"
+
+	/* IPv4/v6 localhost address */
+	IPV4_LOCALHOST = "127.0.0.1"
+	IPV6_LOCALHOST = "::1"
+)
+
+type Sflow struct {
+	Enabled          string
+	Polling_Interval string
+	Agent            string
+}
+
+type SflowCol struct {
+	Ip   string
+	Port string
+	Vrf  string
+}
+
+type SflowIntf struct {
+	Enabled       string
+	Sampling_Rate string
+}
+
+func init() {
+	XlateFuncBind("DbToYang_sflow_xfmr", DbToYang_sflow_xfmr)
+	XlateFuncBind("YangToDb_sflow_xfmr", YangToDb_sflow_xfmr)
+	XlateFuncBind("Subscribe_sflow_xfmr", Subscribe_sflow_xfmr)
+	XlateFuncBind("DbToYang_sflow_collector_xfmr", DbToYang_sflow_collector_xfmr)
+	XlateFuncBind("YangToDb_sflow_collector_xfmr", YangToDb_sflow_collector_xfmr)
+	XlateFuncBind("Subscribe_sflow_collector_xfmr", Subscribe_sflow_collector_xfmr)
+	XlateFuncBind("DbToYangPath_sflow_collector_path_xfmr", DbToYangPath_sflow_collector_path_xfmr)
+	XlateFuncBind("DbToYang_sflow_interface_xfmr", DbToYang_sflow_interface_xfmr)
+	XlateFuncBind("YangToDb_sflow_interface_xfmr", YangToDb_sflow_interface_xfmr)
+	XlateFuncBind("Subscribe_sflow_interface_xfmr", Subscribe_sflow_interface_xfmr)
+	XlateFuncBind("DbToYangPath_sflow_interface_path_xfmr", DbToYangPath_sflow_interface_path_xfmr)
+}
+
+var DbToYangPath_sflow_collector_path_xfmr PathXfmrDbToYangFunc = func(params XfmrDbToYgPathParams) error {
+	log.Info("DbToYangPath_sflow_collector fmr: tbl:", params.tblName)
```

Comment:
Please change all these debug logs into verbose logs -- `log.V(3).Info(...)`

All logs gets redirected to main syslog in the sonic community images. These info logs will flood the syslog with useless data.